src/js/utils/DefaultErrorUtil.tsx

```diff
@@ -0,0 +1,38 @@
+import { i18nMark } from "@lingui/react";
+
+// @ts-ignore
+import {
+  ToastNotification,
+  ToastAppearance
+} from "@extension-kid/toast-notifications";
+
+import container from "#SRC/js/container";
+import {
+  NotificationService,
+  NotificationServiceType
+} from "@extension-kid/notification-service";
+
+const notificationService = container.get(
```

Comment:
you can also type this like: `const notificationService = container.get<NotificationService>(NotificationServiceType);`